src/components/UserFollowersPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import TopBarProgress from 'react-topbar-progress-indicator';
+import PropTypes from 'prop-types';
+import {
+  getAllFollowers,
+} from '../redux/actions/userFollowActions';
+import UserFollowSpan from './common/userFollowSpan';
+
+
+class UserFollowersPage extends Component {
+  componentDidMount() {
+    const { auth: { isVerified } } = this.props;
+    if (isVerified) this.props.getAllFollowers();
+  }
+
+  render() {
+    let followersView;
+    const { userFollow: { isFollowersLoading: isLoading, followers } } = this.props;
+    if (!isLoading && !followers.length) {
```

Comment:
I think it's easier to read this as `!(isLoading || followers.length)`. It also applies from De Morgan's laws.